### PR TITLE
ci: Fix logic to include GPU_FLAG

### PR DESCRIPTION
**Pull request by seemethere**
GPU_FLAG wasn't getting set properly (see [logs](https://github.com/pytorch/audio/actions/runs/4733965450/jobs/8402221728?pr=3271#step:10:160))

### CODE REVIEW

File: .github/scripts/run_docker_with_env_secrets.py [hunk 1]
```diff
@@ -78,4 +78,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
\ No newline at end of file
+    main()
```
Comment (seemethere) [inline .github/scripts/run_docker_with_env_secrets.py:81]: There wasn't a newline here? o.O

**Review by jeanschmidt — APPROVED** [review]

File: .github/scripts/run_docker_with_env_secrets.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def main():
         --shm-size=2g \
         --tty \
         --ulimit stack=10485760:83886080 \
-        { os.environ.get('GPU_FLAG', '') if os.environ.get('CPU_FLAG') else '' } \
+        { os.environ.get('GPU_FLAG', '') } \
```
Comment (seemethere) [inline .github/scripts/run_docker_with_env_secrets.py:66]: in the event that GPU_FLAG isn't set this should just return a blank string

**Review by weiwangmeta — COMMENTED** [review]
Do we have a need to run os.environ.get('CPU_FLAG') on some non-GPU instance?

**Comment by seemethere** [comment]
> Do we have a need to run os.environ.get('CPU_FLAG') on some non-GPU instance?

CPU_FLAG doesn't exist, not entirely sure where that came from

**Review by atalman — APPROVED** [review]

**Comment by jeanschmidt** [comment]
@seemethere `CPU_FLAG` seems a clear typo of `GPU_FLAG`

**Review by weiwangmeta — APPROVED** [review]
Thanks for the quick fix!

**Pull request merged**